website/conferences/message.py

```diff
@@ -21,17 +21,13 @@
 SPF_PASS_VALUES = ['Pass', 'Neutral']
 
 ANGLE_BRACKETS_REGEX = re.compile(r'<(.*?)>')
-ROUTE_REGEX = re.compile(
-    r'''
+BASE_REGEX = r'''
         (?P<test>test-)?
         (?P<meeting>\w*?)
         -
-        (?P<category>poster|talk)
+        (?P<category>{0})
```

Comment:
Minor: Using a keyword instead of positional argument might improve readability here and will also be less error-prone.

```python
(?P<category>{allowed_types})

# ...
.format(allowed_types=self.find_allowed_types, ...)
```